docs/msiv1_token_revocation.md

```diff
@@ -0,0 +1,74 @@
+# MSAL Support for MSI v1 Token Revocation and Capabilities
+
+This specification describes **two optional parameters** that MSAL can send to the **MSI v1 endpoint** (e.g., `http://169.254.169.254/metadata/identity/oauth2/token`) when acquiring tokens via **Managed Identity**. These parameters empower developers to control **token revocation** and specify **client capabilities** that might alter the token issuance behavior by Azure AD (ESTS).
+
+---
+
+## Overview
```

Comment:
Needs more details on the E2E, i.e. how does token revocation happen? i.e. what script do I run as a tenant admin?